backend/app/tasks/llm.py

```
@@ -69,9 +109,21 @@ def llm_routing_worker(
     )
     async def _execute():
         async with get_session(tenant_id=model.tenant_id) as session:
-            return await route_request(model, session=session)
+            return await route_request(model, session=session, force_failure=force_failure)
 
-    return _run_async(_execute)
+    try:
+        return _run_async(_execute)
+    except Exception as exc:
```

Ruff (0.14.11)

Do not catch blind exception: `Exception`

(BLE001)